argilla/docs/how_to_guides/making_most_of_markdown.md

```diff
@@ -0,0 +1,177 @@
+# Making the most of Markdown
```

Comment:
```suggestion
# Use Markdown for displaying chats and media content
```

Reply:
Probably not the best because we are mixing two things: media and chat, but I'd like to keep titles very explicit so they are easier to find

Reply:
alternatives:

Use Markdown to format rich content
Use Markdown and HTML to format content